### PR TITLE
feat: case insensitive track search

### PR DESCRIPTION
**Pull request by elgiovanno**
Hey. Saw your comment on the forum about this. I had already done this so I thought I'd submit it for review even if you choose to make it a config option. If you like it, I can also add the config option on my end. RK for life!

### CODE REVIEW

**Comment by gwatcha** [comment]
Thanks! Now that I think about it I think most users would prefer having case insensitive search, so I won't put in a configuration option.

File: internal/custom_actions/utils.lua [hunk 1]
```diff
@@ -129,9 +129,10 @@ function utils.getMatchedTrack(search_name, forward)
       end
     else
       local _, current_name = reaper.GetTrackName(track, "")
-      local has_no_name = current_name:match("Track ([0-9]+)", 1)
```
Comment (gwatcha) [inline internal/custom_actions/utils.lua:132]: I think it would be better to just keep this line instead of needing to call `string.lower` twice

Comment (elgiovanno) [reply internal/custom_actions/utils.lua:132]: Hi! Just to be clear, you mean reverting just that line to what it was before string.lower?

Comment (gwatcha) [reply internal/custom_actions/utils.lua:132]:
```
 else
      local _, current_name = reaper.GetTrackName(track, "")
      local has_no_name = current_name:match("Track ([0-9]+)", 1)
      current_name = current_name:lower()
      tracks_searched = tracks_searched + 1
      if not has_no_name and current_name:match(search_name:lower()) then
        return track
      end
```

Comment (gwatcha) [reply internal/custom_actions/utils.lua:132]: here's what I meant

Comment (elgiovanno) [reply internal/custom_actions/utils.lua:132]: Got it. That's what I thought. And that makes sense. No need to make it lowercase. I'll get that committed and push.

**Pull request merged**